spring-integration-zookeeper/src/test/java/org/springframework/integration/zookeeper/metadata/ZookeeperMetadataStoreTests.java

```diff
@@ -49,10 +38,21 @@
 import org.springframework.integration.support.utils.IntegrationUtils;
 import org.springframework.integration.zookeeper.ZookeeperTestSupport;
 
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static org.springframework.integration.test.matcher.EqualsResultMatcher.equalsResult;
+import static org.springframework.integration.test.matcher.EventuallyMatcher.eventually;
```

Comment:
Yeah... So, time to remove `<suppress files="[\\/]test[\\/]" checks="ImportOrder" />` for tests! 😄 

There is an idea to move to the https://github.com/spring-io/spring-javaformat, but it doesn't look like they support imports reorder...

Reply:
OK. So, the Checkstyle for import orders in tests as well has been pushed to master: https://github.com/spring-projects/spring-integration/commit/648aeb8e966c63e49a42ce6cb79cafab56ae73d7

Therefore you don't have choice unless revert all the import order changes 😄